Sources/Build/BuildPlan.swift

```diff
@@ -351,6 +351,10 @@ public final class ClangTargetBuildDescription {
         if clangTarget.isCXX {
             args += self.buildParameters.flags.cxxCompilerFlags
         }
+
+        if target.type == .library && !target.dependsOnAnySystemModules {
```

Comment:
We should probably add `buildParameters.triple.isDarwin()`

Reply:
Yep, looks like it